apps/keira/src/app/features/item/item-template/item-preview.service.ts

```diff
@@ -546,9 +546,10 @@ export class ItemPreviewService {
       const type = Number(itemTemplate['stat_type' + i]);
       const qty = Number(itemTemplate['stat_value' + i]);
 
-      if (!qty || !type || type <= 0) {
+      if (!qty || (!type && type !== 0) || type < 0) {
         continue;
       }
+
```

Comment:
missing new unit tests... whenever we fix a bug, ideally we should add a new unit test case that would catch that bug